src/StripeTests/Services/UsageRecordSummaries/UsageRecordSummaryServiceTest.cs

```diff
@@ -16,13 +17,17 @@ public UsageRecordSummaryServiceTest()
         {
             this.service = new UsageRecordSummaryService();
 
-            this.listOptions = new UsageRecordSummaryListOptions();
+            this.listOptions = new UsageRecordSummaryListOptions()
+            {
+                Limit = 1,
+            };
```

Comment:
why did you have to add this?

Reply:
I didn't have to -- it's just the first test I played around with in this PR and I added this to ensure that query parameters weren't part of the `AbsoluteUrl` when asserting the URL. I left it since it's consistent with other list tests.

Reply:
Ah perfect! Thanks for clarifying, let's merge and have fun rebasing :p